prefect_databricks/flows.py

```diff
@@ -312,6 +305,194 @@ def jobs_runs_submit_and_wait_for_completion_flow(notebook_path, **base_paramete
         )
 
 
+@flow(
+    name="Submit existing job runs and wait for completion",
+    description=(
+        "Triggers a Databricks jobs runs and waits for the "
+        "triggered runs to complete."
+    ),
+)
+async def jobs_runs_submit_by_id_and_wait_for_completion(
+    databricks_credentials: DatabricksCredentials,
+    job_id: int = None,
+    idempotency_token: Optional[str] = None,
+    jar_params: Optional[List[str]] = None,
+    max_wait_seconds: int = 900,
+    poll_frequency_seconds: int = 10,
+    notebook_params: Optional[Dict] = None,
+    python_params: Optional[List[str]] = None,
+    spark_submit_params: Optional[List[str]] = None,
+    python_named_params: Optional[Dict] = None,
+    pipeline_params: Optional[str] = None,
+    sql_params: Optional[Dict] = None,
+    dbt_commands: Optional[List] = None,
+    **jobs_runs_submit_kwargs: Dict[str, Any],
+) -> Dict:
+    """flow that triggers an existing job and waits for its completion
+
+    Args:
+        databricks_credentials (DatabricksCredentials):
+            Credentials to use for authentication with Databricks.
```

Comment:
I don't think the typing in the docstring is necessary since the function has type hints
```suggestion
        databricks_credentials: Credentials to use for authentication with Databricks.
```